bravado/http_client.py

```diff
@@ -197,6 +197,9 @@ def __init__(self, session, request):
         self.session = session
         self.request = request
 
+    def result(self, *args, **kwargs):
```

Comment:
So, in `operation.py` [L156](https://github.com/Yelp/swagger-py/blob/swagger2.0/bravado/mapping/operation.py#L156) instead of returning the `__call()__` as:
```python
       return HTTPFuture(self.swagger_spec.http_client, request, response_future)
```
should the signature change to : 
```python
    return self.swagger_spec.http_client.request(request, response_future)
```

Which means, `http_client`'s `request()` should now return back a `HttpFuture` object?